samples/bluetooth/hci_usb/prj.conf

```diff
@@ -1,16 +1,16 @@
-CONFIG_STDOUT_CONSOLE=y
-CONFIG_GPIO=y
-CONFIG_SERIAL=y
-CONFIG_UART_INTERRUPT_DRIVEN=y
-CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
-
```

Comment:
okay, it does not fix a bug and is a problematic change, merged without maintainer approval, this patch must be reverted.

Reply:
What is the problem with disabling console?